software/externals/cmake/externals.cmake

```diff
@@ -139,8 +139,12 @@ set(flycapture_external_args
   SOURCE_DIR ${source_prefix}/../drivers/flycapture
   )
 
-set(iris_url ssh://git@github.com/openhumanoids/iris-distro.git)
-set(iris_revision b278a85ba4b3b3da3af41feb39c13e4150d12d98)
+set(swigmake_url ssh://git@github.com/rdeits/swigmake.git)
```

Comment:
swigmake should also be added to the .gitignore in that folder (I dont have permissions to add that commit since it's a personal fork)